_source/logzio_collections/_security-sources/cynet.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Ship logs from Cynet
+logo:
+  logofile: cynet.png
+  orientation: vertical
+data-source: Cynet
+templates: ["network-device-filebeat"]
+contributors:
+  - nshishkin
+shipping-tags:
+  - firewalls
+order: 1380
+---
+[Cynet](https://www.cynet.com/platform/) is a cybersecurity asset management platform. This topic describes how to send system logs from your Cynet platform to Logz.io.
+
+**Before you begin, you'll need**: 
+
+* An active Cynet license
+* Cynet login credentials 
+* An active account with Logz.io
+* Filebeat 7 installed on your machine
+* Root priveleges on your machines 
+
+<div class="tasklist">
+
+
+##### Configure Cynet to send syslog notifications to a remote Syslog server running Filebeat
+
+1. On your Cynet web interface, go to **Setting > Advanced**.
+![Advanced options](https://dytvr9ot2sszz.cloudfront.net/logz-docs/Cynet/cynet1.png)
+2. Select the box beside **Send Audit Records to SIEM**.
+3. Go to **Configuration > SIEM settings** and enable the following configuration:
+![SIEM settings before](https://dytvr9ot2sszz.cloudfront.net/logz-docs/Cynet/cynet2before.jpg)
+   * **UDP**
+   * **IP** - public IP adrress of your syslog server
+   * **Port** - port that is configured on your syslog server
+4. Press **Add**. The added IP and port will appear on the screen.
+![SIEM settings after](https://dytvr9ot2sszz.cloudfront.net/logz-docs/Cynet/cynet2after.jpg)
+  
+<!-- info-box-start:info -->
+These instructions are based on UDP. If you want to use TCP, make sure your syslog server configuration is aligned with this.
+{:.info-box.note}
+<!-- info-box-end -->
+
+{% include log-shipping/certificate.md %}
+
+
+##### Configure Filebeat
+
+1. Paste the following into the inputs section of the Filebeat configuration file:
+
+{% include log-shipping/syslog-filebeat.md %}
+ 
+   * 0.0.0.0.:5044 is the default address and port of the Filebeat server. If you use a different address and port, replace the default values with your parameters.
+   * Replace `<<LOG_TYPE>>` with `cynet`.
+   * Replace `<<CODEC-TYPE>>` with `json`.
```

Comment:
@nico-shishkin 
Can we delete this instruction ?
most(if not all ) of our content are based on json codec when we use filebeat, if you can see on older instructions we are just using a filebeat.yml with json codec on the example they are going to use. without asking them to change this section
thanks.